File: SERVICE/BakTraCam.Service.DataContract/BakimModel.cs
using System;

namespace BakTraCam.Service.DataContract
{
    public class BakimModel
    {
        public int Id { get; set; }
        public string Aciklama { get; set; }
        public string Ad { get; set; }
        public DateTime Tarihi { get; set; }
        public int Gerceklestiren1 { get; set; }
        public int Gerceklestiren2 { get; set; }
        public int Gerceklestiren3 { get; set; }
        public int Gerceklestiren4 { get; set; }
        public int Sorumlu1 { get; set; }
        public int Sorumlu2 { get; set; }
        public int Period { get; set; }
        public int Durum { get; set; }
        public int Tip { get; set; }
    }
}

File: BakTraCam.ClientApp/src/app/models/bakim.model.ts
export interface BakimModel {
    Id: number,
    Aciklama: string,
    Ad: string,
    Gerceklestiren1: number,
    Gerceklestiren2: number,
    Gerceklestiren3: number,
    Gerceklestiren4: number,
    Sorumlu1: number,
    Sorumlu2: number,
    Tarihi: Date,
    BaslangicTarihi: Date;
    BitisTarihi: Date;
    Durum: number,
    Period: number,
    Tip: number
}
export interface BakimModelBasic {
    Id: number,
    Aciklama: string,
    Ad: string,
    Gerceklestiren1: string,
    Gerceklestiren2: string,
    Gerceklestiren3: string,
    Gerceklestiren4: string,
    Sorumlu1: string,
    Sorumlu2: string,
    Tarihi: Date,
    Durum: number,
    Period: number,
    Tip: number
}
